dev/ProgressRing/ProgressRing.cpp

```diff
@@ -39,18 +46,40 @@ void ProgressRing::OnApplyTemplate()
 {
     winrt::IControlProtected controlProtected{ *this };
 
-    m_player.set(GetTemplateChildT<winrt::AnimatedVisualPlayer>(s_IndeterminateAnimatedVisualPlayerName, controlProtected));
     m_layoutRoot.set(GetTemplateChildT<winrt::Grid>(s_LayoutRootName, controlProtected));
 
+    m_determinatePlayer.set([this, controlProtected]()
```

Comment:
>[this, controlProtected]() [](start = 28, length = 26)

I don't think this lambda adds anything if you are no longer attaching anything to it.